test/src/unit-cppapi-global-order-writes-remote.cc

```diff
@@ -0,0 +1,504 @@
+/**
+ * @file unit-cppapi-global-order-writes-remote.cc
+ *
+ * @section LICENSE
+ *
+ * The MIT License
+ *
+ * @copyright Copyright (c) 2023 TileDB Inc.
```

Comment:
Remove @copyright Copyright (c) 2016 MIT and Intel Corporation. This is a new file.